lib/fuzzer/main.zig

```diff
@@ -0,0 +1,508 @@
+const std = @import("std");
+const builtin = @import("builtin");
+const mutate = @import("mutate.zig");
+const InputPool = @import("input_pool.zig").InputPool;
+const MemoryMappedList = @import("memory_mapped_list.zig").MemoryMappedList;
+const feature_capture = @import("feature_capture.zig");
+
+const Allocator = std.mem.Allocator;
+const ArrayList = std.ArrayList;
+const ArrayListUnmanaged = std.ArrayListUnmanaged;
+const Options = std.testing.FuzzInputOptions;
+const Prng = std.Random.DefaultPrng;
+const SeenPcsHeader = std.Build.Fuzz.abi.SeenPcsHeader;
+const assert = std.debug.assert;
+const fatal = std.process.fatal;
+
+const Testee = *const fn ([*]const u8, usize) callconv(.C) void;
+
+const InitialFeatureBufferCap = 64;
+
+// currently unused
+export threadlocal var __sancov_lowest_stack: usize = std.math.maxInt(usize);
+
+/// Type for passing slices across extern functions where we can't use zig
+/// types
+pub const Slice = extern struct {
+    ptr: [*]const u8,
+    len: usize,
+
+    pub fn toZig(s: Slice) []const u8 {
+        return s.ptr[0..s.len];
+    }
+
+    pub fn fromZig(s: []const u8) Slice {
+        return .{
+            .ptr = s.ptr,
+            .len = s.len,
+        };
+    }
+};
```

Comment:
This is not used anywhere

Reply:
It is used by the `fuzzer_init` function in `fuzzer.zig`. Arguably `fuzzer.zig` is a better place for this.